src/main/java/uk/gov/hmcts/reform/notificationservice/task/NotificationMessageProcessTask.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.notificationservice.task;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.notificationservice.service.NotificationMessageProcessor;
+
+@Service
+@ConditionalOnProperty(value = "scheduling.task.notifications-consume.enabled", matchIfMissing = true)
+public class NotificationMessageProcessTask {
+
+    private static final Logger log = LoggerFactory.getLogger(NotificationMessageProcessTask.class);
+
+    private final NotificationMessageProcessor notificationMessageProcessor;
+
+    public NotificationMessageProcessTask(
+        NotificationMessageProcessor notificationMessageProcessor
+    ) {
+        this.notificationMessageProcessor = notificationMessageProcessor;
+    }
+
+    @Scheduled(fixedDelayString = "${scheduling.task.notifications-consume.delay}")
+    public void consumeMessages() {
+        log.info("Started the job consuming notification messages");
```

Comment:
i think we usually do `"Started/Finished {task-name} task"` where `{task-name}` is statically defined string with hyphen separated words if necessary, e.g. `consume-notifications` in this case